lib/controller.js

```diff
@@ -39,13 +39,16 @@ function lnCmsController($rootScope, $scope, lnCmsClientService, $urlRouter) {
         //load view
         lnCmsClientService.getView(endpoint, params)
           .then(function(response) {
-            if (response.data.length > 0)
+            if (response.data.length > 0) {
```

Comment:
It's not from staging - http://wp.skaled.moxie-staging.com/wp-json/leean/v1/post?id=1 returns an object:
http://snaps.getmoxied.net/3v2z0T1F0n2L